### PR TITLE
All functionality of Stack in Javascript

### PR DESCRIPTION
**Pull request by pulkitgovrani**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

I have added all the functionality of stack in javascript. And also added it's link in the readme.md file.

### Checklist

- [✓] I've read the contribution guidelines.
- [✓] I've checked the issue list before deciding what to submit.
- [✓] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Comment by pulkitgovrani** [comment]
Please review this @HarshCasper

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
Do the following modifications
* Use Dynamic input for better user friendly interface
* create a count function for counting the total number of elements present at that time
* Display suitable message after each operation

**Review by vatsalkesarwani12 — APPROVED** [review]
LGTM 👍 
Try to make code dynamic

**Comment by pulkitgovrani** [comment]
@siddharth25pandey  sir please check now i have done all the changes.

File: JavaScript/ds/Stack.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+//Removing the topmost element from the stack by pop()
+var removed_element=mystack.pop();
+
+console.log("The element which is popped from the stack is ",removed_element);
+
+//Printing the stack after removing the element using print()
+console.log("The elements present after removing one element in stack are ",mystack.print());
+
+
```
Comment (atarax665) [inline JavaScript/ds/Stack.js:109]: Add sample Input/Output in commented form.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: JavaScript/ds/Stack.js (hunk 1 above)
Comment (pulkitgovrani) [reply JavaScript/ds/Stack.js:109]: I have done all the changes sir please review them.

**Comment by siddharth25pandey** [comment]
@pulkitgovrani Work on point number one of [this](https://github.com/TesseractCoding/NeoAlgo/pull/777#pullrequestreview-474156127) Remove the comments & make it proper functionable for user .

**Comment by pulkitgovrani** [comment]
@siddharth25pandey Sir please check now i have removed the comments from input.

**Review by siddharth25pandey — APPROVED** [review]
@pulkitgovrani , Good to go
Able to merge

**Review by atarax665 — APPROVED** [review]
LGTM :+1:

**Pull request merged**